### ISSUE TITLE
Enable allow-lists for certain paths to unauthenticated users

### ISSUE THREAD

**Issue by f-herceg** [gravitational/teleport#12820]
What would you like Teleport to do?
To allow certain paths for applications that are proxied through Teleport to allow using webhooks. Currently that is not possible as services like Github can't authenticate against Teleport.

What problem does this solve?
Being able to allow a path in Teleport would help because it wouldn't be necessary to create a different ingress entry for the applications which use webhooks which is currently the case.

If a workaround exists, please include it.
Workaround that comes to my mind is to create a separate ingress for the application and allow traffic only on the path for the webhook.

**Comment by tchellomello** [comment]
Yes, that feature would be so cool to expose for instance ArgoCD webhooks on GitHub
